### PR TITLE
[editor]Add new POI in editor

### PR DESCRIPTION
**Pull request by Jean-BaptisteC**
I have again add new type in editor.config

@biodranik I have some questions about this file:

- Is it possible to arrange by alphabetical order foreach types for better visibility ?
-  Order is use in code ? https://github.com/organicmaps/organicmaps/blob/6600f3fb5f35d956eb879bf3d7d2226ea0c838aa/data/editor.config#L157
- Can i add missing group in types ?
- We have missing this :) https://github.com/organicmaps/organicmaps/blob/6600f3fb5f35d956eb879bf3d7d2226ea0c838aa/data/editor.config#L2
- Do you think it's necessary to include field internet for shop ?

### CODE REVIEW

File: data/editor.config [hunk 1]
```diff
@@ -621,10 +627,16 @@
...
       </type>
+      <type id="shop-chocolate" group="shop">
+        <include group="poi" />
+      </type>
       <type id="shop-clothes" group="shop">
         <include group="poi" />
         <include field="internet" />
       </type>
+      <type id="shop-coffee" group="shop">
+        <include group="poi" />
```
Comment (biodranik) [inline data/editor.config:638]: Why are you not adding an internet field? Btw, should it be included in POI by default? What are the exceptions without Internet?

**Comment by Jean-BaptisteC** [comment]
> 1. Aren't types displayed sorted already?

Types have alphabetical order in app, but maybe you can sort by key and by values types in xml
>     2. Need to review the code to remember the reasoning behind it. It was 2016 ) Can you try to check it yourself?
I have find commit https://github.com/organicmaps/organicmaps/commit/424d4d14571dfd154b183cd4820d106344745dd2
>     3. What missing groups do you have in mind?
Some shops have don't shop group in xml (I have check in Wiki) 
- https://wiki.openstreetmap.org/wiki/Tag:shop%3Ddry_cleaning
- https://wiki.openstreetmap.org/wiki/Tag:shop%3Dmassage
>     4. Argh, good catch! I think we can safely replace it to organicmaps or omaps.
Fixed :)
>     5. Some shops have internet. It is VERY useful when you're in roaming or didn't buy a SIM card yet, or exhausted your limits while traveling. And our search easily finds these spots.

Ok to add in all shop but maybe disable on office and POI about education ?
https://wiki.openstreetmap.org/wiki/Key:internet_access

**Comment by biodranik** [comment]
1. Do You mean sorting types in the editor.config? It is possible partially if types have the same priority. Generic building type, for example, should have the lowest priority.
2. I meant the code logic behind it. It starts here: https://github.com/organicmaps/organicmaps/blob/6f67523f87a0c642dc06057a12aa8d6bd0384ece/editor/editor_config.cpp#L109 , you can see where and why is this function used.
3. I don't see that "shop" group is defined. Was it removed if you check in the git history? Why? How is it different from other groups? Did you get how the config works?
4. Why do you want to disable internet for offices and education? If they have a free or guest wifi, then it's ok.
5. Can you please add a separate github check called editor-check.yml that will checkout only the main repo and validate the editor config by running
`xmllint -noout -schema config.xsd editor.config`
?

**Comment by Jean-BaptisteC** [comment]
- I have reorder alphabetticaly amenity types and shops types
- Add internet  field on all shops types
- I have add missing groups on somes types
- I have try to fixed failed CI by replace mapsme by omaps in all files in editor directory
- Of course, but i prefere to do that in separated PR

**Review by biodranik — APPROVED** [review]
Good. Let's
1) add "internet" into every poi in a separate PR (mapsme renaming can be in a separate commit there too).
2) Sort them in a separate PR

**Pull request merged**